gci/views.py

```diff
@@ -14,6 +15,11 @@ def index(request):
     org_id = linked_students[0]['organization_id']
     org_name = linked_students[0]['organization_name']
     s = []
+
+    get_logo()
+
+    s.append('<link rel="shortcut icon" type="image/png" href="favicon.png"/>')
+    s.append('<img href="images/org_logo.png">')
```

Comment:
Should be `src` instead of `href`
Are you sleeping again?

Also, `alt` attribute with the org name would be nice to have here.

Reply:
Let’s just leave it as I’m a bit rusty with my HTML